apertium-uzb-kaa.uzb-kaa.dix

```diff
@@ -1,159 +1,159 @@
+    <sdef n="ind"   c="Indefinite"/> 
+    <sdef n="itg"   c="Interrogative"/> 
+    <sdef n="qnt"   c="Quantifier"/> 
+    <sdef n="neg"   c="Negative"/> 
+    <sdef n="ref"   c="Reflexive"/> 
+    <sdef n="ord"   c=""/> 
+    <sdef n="caus"  c=""/> 
+    <sdef n="pass"  c=""/> 
+    <sdef n="coop"  c=""/> 
+    <sdef n="prc_perf"  c=""/> 
+    <sdef n="ger"   c=""/> 
+    <sdef n="gpr_past"  c=""/> 
+    <sdef n="gpr_pot"   c=""/> 
+    <sdef n="gpr_ppot"  c=""/> 
+    <sdef n="iv"    c="Intransitive"/> 
+    <sdef n="tv"    c="Transitive"/> 
+    <sdef n="TD"    c="Temporary tag"/> 
+    <sdef n="p1"    c=""/> 
+    <sdef n="p2"    c=""/> 
+    <sdef n="p3"    c=""/> 
+    <sdef n="frm"   c=""/> 
+    <sdef n="qst"   c="Question modal (clitic)"/> 
+    <sdef n="emph"  c="Emphasising modal"/> 
+    <sdef n="mod_ass"   c="Assertive modal (clitic)"/> 
+    <sdef n="mod_ind"   c="Indefinite modal (clitic)"/> 
+    <sdef n="mod"   c="Modal verbs"/> 
+    <sdef n="sent"  c=""/> 
+    <sdef n="guio"  c=""/> 
+    <sdef n="cm"    c=""/> 
+    <sdef n="apos"  c=""/> 
+    <sdef n="abl"   c=""/> 
+    <sdef n="nom"   c=""/> 
+    <sdef n="gen"   c=""/> 
+    <sdef n="rquot"     c=""/> 
+    <sdef n="lquot"     c=""/> 
+    <sdef n="rpar"  c=""/> 
+    <sdef n="lpar"  c=""/> 
     <sdef n="infl" /> 
     <sdef n="paren" /> 
     <sdef n="refl" /> 
     <sdef n="ca" /> 
     <sdef n="cs" /> 
     <sdef n="prep" /> 
-    <sdef n="REmOvE" 	c=""/> 
+    <sdef n="REmOvE"    c=""/> 
   </sdefs>
   <section id="main" type="standard">
 
       <!-- SECTION: Common-->
 
-<e>       <p><l>abadiyan<s n="adv"/></l>               <r>abadiyan<s n="adv"/></r></p></e>
-<e>       <p><l>abadiya<s n="adj"/></l>                <r>abadiya<s n="adj"/></r></p></e>
+<!-- <e>       <p><l>abadiyan<s n="adv"/></l>               <r>abadiyan<s n="adv"/></r></p></e> -->
+<!-- <e>       <p><l>abadiya<s n="adj"/></l>                <r>abadiya<s n="adj"/></r></p></e>
 <e>       <p><l>abadiyat<s n="n"/></l>                 <r>abadiyat<s n="n"/></r></p></e>
 <e>       <p><l>abadiylash<s n="n"/></l>               <r>abadiylash<s n="n"/></r></p></e>
 <e>       <p><l>abadiylashtir<s n="v"/><s n="tv"/></l> <r>abadiylashtir<s n="v"/><s n="tv"/></r></p></e>
-<e>       <p><l>abadiylik<s n="n"/></l>                <r>abadiylik<s n="n"/></r></p></e>
-<e>       <p><l>abadiy<s n="adj"/></l>                 <r>abadiy<s n="adj"/></r></p></e>
-<e>       <p><l>abad<s n="adv"/></l>                   <r>abad<s n="adv"/></r></p></e>
-<e>       <p><l>abad<s n="n"/></l>                     <r>abad<s n="n"/></r></p></e>
-<e>       <p><l>abajur<s n="n"/></l>                   <r>abajur<s n="n"/></r></p></e>
-<e>       <p><l>abas<s n="adj"/></l>                   <r>abas<s n="adj"/></r></p></e>
-<e>       <p><l>abdol<s n="adj"/></l>                  <r>abdol<s n="adj"/></r></p></e>
+<e>       <p><l>abadiylik<s n="n"/></l>                <r>abadiylik<s n="n"/></r></p></e> -->
```

Comment:
You should comment out words individually, not in blocks.

Also, I believe these words were scraped from a dictionary (@IlnarSelimcan?)?  Is it true that none of them can ever translate this way?

Reply:
Because the pair [abadiylik - máńgilik] is existing already

Reply:
The bilingual dictionary should contain **all possible mappings between words**.

You didn't answer my question, but if the words *can* translate this way, then you should *not* remove the entries.

Reply:
Also, now you've deleted them instead of commenting them out.  If you're unsure, you can just comment out words, but do them individually, not in blocks.

Reply:
[abadiy - abadiy] is wrong. The right one will be [abadiy-máńgilik]. And it is existing.  So I deleted the wrong one.

Reply:
Are all of the words that you removed entirely impossible translations?

Reply:
Yes, they either exist words or are [uzb-uzb] words so I removed them all.

Reply:
What do you mean that they exist?  Do you mean they're duplicates?

Reply:
yes, I mean they are true duplicates

Reply:
Sorry haven't replied to this tread in a timely fashion. I believe those uzb-uzb entries were created automatically as templates for qareken.